go/service/delegate_ui.go

```diff
@@ -47,3 +47,14 @@ func (d *DelegateUICtlHandler) RegisterUpdateUI(_ context.Context) error {
 	d.G().UIRouter.SetUI(d.id, libkb.UpdateUIKind)
 	return nil
 }
+
+func (d *DelegateUICtlHandler) RegisterRekeyUI(_ context.Context) error {
+	d.G().UIRouter.SetUI(d.id, libkb.RekeyUIKind)
+
+	// Let Gregor related code know that a RekeyUI client
```

Comment:
IF we're in CLI mode, we never get a NewRekeyUIHandler registered.  And therefore, we won't process RekeyNeeded gregor notifications. And therefore we won't output to the error log that a rekey is needed....